scripts/e2e/generate-e2e-spec.sh

```diff
@@ -0,0 +1,24 @@
+echo "Creating e2e spec files using 'generate-yamls.sh'..."
+
+# first set the appropriate envoironment variables gor generate-yamls.sh
+export VSPHERE_SERVER='$VSPHERE_SERVER'
+export VSPHERE_USERNAME='$VSPHERE_USERNAME'
+export VSPHERE_PASSWORD='$VSPHERE_PASSWORD'
+export VSPHERE_DATACENTER='SDDC-Datacenter'
+export VSPHERE_DATASTORE='WorkloadDatastore'
+export VSPHERE_NETWORK='sddc-cgw-network-5'
+export VSPHERE_RESOURCE_POOL='clusterapi'
+export VSPHERE_FOLDER='clusterapi'
+export VSPHERE_TEMPLATE='ubuntu-1804-kube-v1.13.6'
+
+
+# run the script to generate the yaml files
+sh ../../../hack/generate-yaml.sh -f -c 'test1' -m '$VSPHERE_CONTROLLER_VERSION' \
+    -o scripts/e2e/bootstrap_job/spec
+
+# rename the file from *.yaml to *.template
+for f in spec/*.yaml
+do
+    mv $f ${f%.*}.template
```

Comment:
Hi @ykakarap,

I don't understand what this is doing.

Reply:
Nevermind. I realize what you did. Still, we shouldn't need these once we move to using the image.